consulnameresolver/src/test/java/io/datanerds/grpc/ConsulNameResolverProviderTest.java

```diff
@@ -0,0 +1,104 @@
+package io.datanerds.grpc;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.net.URI;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.*;
+
+public class ConsulNameResolverProviderTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Test
```

Comment:
I've seen other test classes below omit the `test` prefix, probably the test methods in this class should be renamed to avoid the superfluous prefix.